### PR TITLE
docs: adds a small description of what a slice is and when an image p…

### PR DESCRIPTION
**Pull request by nikhedonia**
Have a look at the readme, let me know if I should add sth.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/70320e5432f2333bcf3ce27eada91cd56fe1a997/dextrose-test/index.html

File: packages/slice/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Slice
+
+Slices are templates for articles. A slice lays out a set of articles according
```
Comment (craigbilner) [inline packages/slice/README.md]: not articles per se, they'll predominantly be used for "article cards"

File: packages/slice/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+to template.
```
Comment (craigbilner) [inline packages/slice/README.md]: would be good to get the template names and examples/point them to the stories?

File: packages/slice/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Image Placeholder Rules
+
+If an article has no image available (either as the lead image or generated from
+the associated video), display a new generic Times/ ST logo, NOT one that is
+specific to the publication (Times/ Sun Times)
```
Comment (craigbilner) [inline packages/slice/README.md]: The Times/ The Sunday Times

File: packages/slice/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+This should apply in the following scenarios when an image is in a slice:
+
+* author profile list page
```
Comment (craigbilner) [inline packages/slice/README.md]: these don't use slices?

**Pull request merged**